app/graphql/types/model_type.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
-class Types::ModelType < Types::BaseObject
-  implements Types::DoiItem
+class ModelType < BaseObject
+  implements DoiItem
```

Comment:
I think the previous version was `implements ::DoiItem` but i'm unsure if that matter much